Sample/SampleForageSDK/Sections/CreatePayment/CreatePaymentView.swift

```diff
@@ -41,62 +41,51 @@ class CreatePaymentView: UIView {
 
     private let snapTextField: UITextField = {
         let tf = UITextField()
-        tf.placeholder = "Snap amount"
+        tf.placeholder = "SNAP amount"
         tf.translatesAutoresizingMaskIntoConstraints = false
         tf.borderStyle = .roundedRect
         tf.accessibilityIdentifier = "tf_snap_amount"
         tf.isAccessibilityElement = true
         return tf
     }()
 
-    private let nonSnapTextField: UITextField = {
+    private let ebtCashTextField: UITextField = {
         let tf = UITextField()
-        tf.placeholder = "Non Snap amount"
+        tf.placeholder = "EBT Cash amount"
         tf.translatesAutoresizingMaskIntoConstraints = false
         tf.borderStyle = .roundedRect
         tf.accessibilityIdentifier = "tf_non_snap_amount"
```

Comment:
While we're at it want to change this to `tf_cash_amount`

Reply:
79c54a1174210eb6ec3d4d77d2d72e319d55cfee